src/gui/entry/EntryModel.cpp

```diff
@@ -145,17 +206,139 @@ QVariant EntryModel::data(const QModelIndex& index, int role) const
+            /*
+             * Display usernames hidden or visible according to current state
+             * of 'Hide Usernames' setting
+             */
+            if (m_hideUsernames) {
+                result = EntryModel::HiddenContent;
+            }
+            else {
+                //result = entry->username();
+                result = entry->resolveMultiplePlaceholders(entry->username());
+            }
             if (attr->isReference(EntryAttributes::UserNameKey)) {
                 result.prepend(tr("Ref: ","Reference abbreviation"));
             }
             return result;
+        case Password:
+            /*
+             * Display passwords hidden or visible according to current state
+             * of 'Hide Passwords' setting
+             */
+            if (m_hidePasswords) {
+                result = EntryModel::HiddenContent;
+            }
+            else {
+                //result = entry->resolveMultiplePlaceholders(entry->password());
+                result = entry->password();
+            }
+            if (attr->isReference(EntryAttributes::PasswordKey)) {
+                result.prepend(tr("Ref: ","Reference abbreviation"));
+            }
+            return result;
         case Url:
+            //result = entry->resolveMultiplePlaceholders(entry->displayUrl());
             result = entry->displayUrl();
             if (attr->isReference(EntryAttributes::URLKey)) {
                 result.prepend(tr("Ref: ","Reference abbreviation"));
             }
             return result;
+        case Notes:
+            /*
+             * Display only first line of notes in simplified format like
+             * KeePassX does
+             */
+            //result = entry->resolveMultiplePlaceholders(entry->notes().section("\n", 0, 0).simplified());
+            result = entry->notes().section("\n", 0, 0).simplified();
+            if (attr->isReference(EntryAttributes::NotesKey)) {
+                result.prepend(tr("Ref: ","Reference abbreviation"));
+            }
+            return result;
+        case Expires:
+            /*
+             * Display either date of expiry or 'Never' like KeePassX does
+             */
+            result = entry->timeInfo().expires() ? entry->timeInfo().expiryTime().toLocalTime().toString(EntryModel::DateFormat) : tr("Never");
+            return result;
+        case Created:
+            result = entry->timeInfo().creationTime().toLocalTime().toString(EntryModel::DateFormat);
+            return result;
+        case Modified:
+            result = entry->timeInfo().lastModificationTime().toLocalTime().toString(EntryModel::DateFormat);
+            return result;
+        case Accessed:
+            result = entry->timeInfo().lastAccessTime().toLocalTime().toString(EntryModel::DateFormat);
+            return result;
+        case Attachments:
+            /*
+             * Display comma-separated list of attachments
```

Comment:
Instead of a list of attachments I recommend either just writing "YES"/"NO" or adding a paperclip icon.

Reply:
ooooh good idea!

Reply:
I like the paperclip

Reply:
Ok, I'll add this column!